### ISSUE TITLE
Fix dates for all posts

### ISSUE THREAD

**Issue by matiasandina** [open-neuroscience/open-neuroscience.github.io#9]
Posts without date will show as January 1st 0001, either assign correct/current date or remove date

**Comment by amchagas** [comment]
How about just adding the current date for the ones with jan 1st 0001? and then changing it everytime it gets changed?

this way at least all posts have a sensible date and readers will know what was the last time that post was updated

**Comment by matiasandina** [comment]
Yeah this is something I normally do automatically while compiling .Rmd files, it has a nice feature that you can do `r Sys.time()` on the date field and will put the correct date each time it renders. 

Let's switch everything to current and then remember to keep updating.
